### ISSUE TITLE
Export apollo-client and graphql-tag from react-apollo

### ISSUE THREAD

**Issue by helfer** [apollographql/react-apollo#428]
This would make getting started easier, people could just `npm install react-apollo` and would be ready to roll.

**Comment by jbaxleyiii** [comment]
@helfer I'm super open to this idea! Would it be confusing docs wise?

`import { Client, ApolloProvider, createNetworkInterface } from "react-apollo"`

or even

```
import { ApolloProvider } from "react-apollo";

<ApolloProvider createClient={clientOptions} />
```

**Comment by stubailo** [comment]
Would this hurt people's ability to pick which version of Apollo client they want? Not sure if that is something we desire.

**Comment by jbaxleyiii** [comment]
@stubailo I was wondering the same thing. They could certainly install AC and use it directly (or with RA) if we set a version range. I could go either way.

Whatever ya'll want 👍 🌮 🌮 🌮

**Comment by stubailo** [comment]
Also I think we shouldn't hide the constructor inside something like provider since many packages will require a client instance or some other thing you wouldn't easily be able to get from this method.

**Comment by JacobSoderblom** [comment]
Hi!

Is this related to the discussion about separate the DOM/SSR logic from Apollo in [apollographql/react-apollo#372](https://github.com/apollographql/react-apollo/pull/372)?

I would love to get Apollo to work with `inferno` without the  `inferno-compat` package. 😄

**Comment by calebmer** [comment]
@JacobSoderblom the ideal scenario is that most of the UI logic for Apollo is inside `apollo-client` and not the view integration layers 😊

An `inferno-apollo` project would probably be the best option. If there is anything that the core team can do with `apollo-client` to make that easier, let us know 👍

**Comment by JacobSoderblom** [comment]
@calebmer Thank you for the answer! 😊

You're totally right about that scenario. I have started an `inferno-apollo` project at the `inferno` repo and are almost done, just the server parts left and some tests, but all the work has been to read your guys code then try to make it work with `inferno`, so even if I don't like to copy others code it has been some of that 😞

I tried to break out the UI logic as much as possible to their own files and classes. It would be lovely if you guys would like to take a look at the work and give some feedback and maybe we could work together to break out the UI logic of `react-apollo` to `apollo-client`. 😊

[inferno-apollo](https://github.com/JacobSoderblom/inferno/tree/inferno-apollo/packages/inferno-apollo/src)
